### PR TITLE
[typo](doc) 字段描述与建表sql中的不一致

### PR DESCRIPTION
**Pull request by UnicornLee**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-TABLE.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ distribution_desc
     
     `ENGINE=olap`
     
-* `key_desc`
+* `keys_type`
```
Comment (carlvinhust2012) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-TABLE.md:178]: the English docs has the same problem, please fix them together.

Comment (UnicornLee) [reply docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-TABLE.md:178]: The problem has been fixed both in EN and ZH docs

**Review by luzhijing — APPROVED** [review]
LGTM

**Comment by luzhijing** [comment]
run P0

**Pull request merged**